invoicing_app/management/commands/create_dummy_data.py

```diff
@@ -0,0 +1,174 @@
+            default=False,
+            help='Clean db'
+        ),
+        make_option(
+            '--quantity',
+            dest='quantity',
+            type='int',
+            help='Enter a quantity'
+        ),
+    )
+
+    def handle(self, **options):
+        if options['quantity']:
+            self.quantity = options['quantity']
+        else:
+            raise CommandError('Please, make sure to enter a number of objects to create: --quantity=X')
+
+        if options['clean_db']:
+            self.delete_all_models()
+
+        today = dt.today()
+        prev_date = dt(today.year, today.month, 1) - relativedelta(months=1)
+        self.date_in = str(dt(today.year, prev_date.month, 12))
+        self.date_out = str(dt(2015, 05, 22, 0, 0))
+
+        self.user_test = UserFactory.create(
+            first_name='user_test'
+        )
+
+        self.insert_columns_in_data_base(self.quantity, self.date_in, self.user_test)
+        self.insert_series_event(self.user_test)
+
+        print('Objects created ok!')
+
+    def delete_all_models(self):
+        self.delete_model_data(Order)
+        self.delete_model_data(PaymentOptions)
+        self.delete_model_data(Event)
+        self.delete_model_data(User)
+        print("Database cleaned")
+
+    def delete_model_data(self, model_class):
+        cursor = connection.cursor()
+        cursor.execute("SET FOREIGN_KEY_CHECKS=0;")
+        table_name = model_class._meta.db_table
+        sql = "DELETE FROM %s WHERE 1;" % (table_name,)
+        cursor.execute(sql)
+        cursor.execute("SET FOREIGN_KEY_CHECKS=1;")
+
+    def insert_columns_in_data_base(self, quantity, date, user_test):
+        event_list = []
+
+        for index in range(quantity):
+            event = EventFactory.create(
+                event_name='event_{}'.format(index),
+                is_series_parent=False,
+                series=False,
+                user=user_test,
+                currency='ARS',
+                repeat_schedule=''
+            )
+            event_list.append(event)
+
+        for index in range(quantity):
+            pay_opt = PaymentOptionsFactory.create(
+                epp_country='AR',
+                accept_eventbrite=True,
+                event=event_list[index],
+                epp_name_on_account='name_test',
+                epp_address1='address1',
+                epp_address2='addres2',
+                epp_zip='5500',
+                epp_city='city_test',
+                epp_state='state_test',
+                epp_tax_identifier=''.join(random.choice(string.lowercase) for x in range(random.randint(11, 12)))
+            )
+
+        for index in range(quantity):
+            order = OrderFactory.create(
+                status=100,
+                pp_date=date,
+                changed=date,
+                event=event_list[index],
+                mg_fee=5.1,
+                gross=1.1,
+                eb_tax=1.1,
+            )
```

Comment:
You can avoid using loops by using the Factory `create_batch` method and using size=quantity
see https://factoryboy.readthedocs.io/en/latest/reference.html#factory.create_batch

Reply:
We tried to use create_batch but had some problems with the object's id so, for do it faster, decided apply loops.

Reply:
I see, in that case you could merge those loops into only one.

Reply:
You're right! I'll fix it!